CMakeLists.txt

```diff
@@ -810,14 +822,9 @@ foreach(sourcefile ${C_TEST_EXES})
     add_test(NAME ${exename} COMMAND ${exename}${ARTIFACT_SUFFIX})
     add_dependencies(check ${exename}${ARTIFACT_SUFFIX})
 endforeach(sourcefile ${C_TEST_EXES})
-add_subdirectory(tools)
+endif()
 
-# Installation and packaging for Linux
-if(NOT WIN32)
```

Comment:
Ok, but what the condition should be?

We cannot check nicely if the `CMAKE_INSTALL_PREFIX` was set by the user, because this var is never empty (has some OS-specific defaults).

How about `SNAPPY_INSTALL_ON_WINDOWS`?

@ruslo any ideas?